lib/jsdom/living/geometry/DOMMatrix.webidl

```diff
@@ -0,0 +1,71 @@
+// https://www.w3.org/TR/geometry-1/#dommatrix
```

Comment:
Always implement the latest editor's draft: https://drafts.fxtf.org/geometry/#dommatrix